internal/app/go-filecoin/node/node.go

```diff
@@ -415,8 +431,14 @@ func (node *Node) setupStorageMining(ctx context.Context) error {
 		return err
 	}
 
+	postProofType, sealProofType, err := registeredProofsFromSectorSize(status.SectorSize)
+	if err != nil {
+		return err
+	}
+
 	sectorBuilder, err := sectorbuilder.New(&sectorbuilder.Config{
-		SectorSize:    status.SectorSize,
+		PoStProofType: postProofType,
+		SealProofType: sealProofType,
```

Comment:
Not a problem for today, but in the future the sectorbuilder will need to support dynamic proof types: per-sector seal proof and per-PoSt proofs.